cves/CVE-2005-2088.yml

```diff
@@ -31,7 +31,7 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: The HTTP Server when acting as an HTTP proxy allowed remote attackers to corrupt the web cache and bypass the web fire wall protection. This allowed for XSS attacks and was a gateway for attackers to send malicious data to the remote server to be accepted as an HTTP request. (Http Request Smuggling)
```

Comment:
This is good, but it needs to go deeper. Assume your reader hasn't taken 331. Explain some of the jargon. Your readers don't know HTTPD, so explain it to them.